app/assets/javascripts/welcome.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
@snugthejoiner I'm a fan of deleting files you will not use. Prepare for me to start marking them.